tests/DocScan/Session/Retrieve/StaticLivenessResourceResponseTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Yoti\Test\DocScan\Session\Retrieve;
+
+use Yoti\DocScan\Session\Retrieve\MediaResponse;
+use Yoti\DocScan\Session\Retrieve\StaticLivenessResourceResponse;
+use Yoti\Test\TestCase;
+
+/**
+ * @coversDefaultClass \Yoti\DocScan\Session\Retrieve\StaticLivenessResourceResponse
+ */
+class StaticLivenessResourceResponseTest extends TestCase
+{
+    private const SOME_LIVENESS_TYPE = 'someLivenessType';
+
+    /**
+     * @test
+     * @covers ::__construct
+     * @covers ::getLivenessType
+     * @covers ::getImage
+     */
+    public function shouldBuildCorrectly()
+    {
+        $input = [
+            'liveness_type' => self::SOME_LIVENESS_TYPE,
+            'image' => [
+                'media' => [
+                    'id' => '349c30c35dc34c',
+                    'type' => 'IMAGE',
+                    'created' => '2021-06-11T11:39:24Z',
+                    'last_updated' => '2021-06-11T11:39:24Z',
+                ]
+            ],
+
+        ];
+
+        $result = new StaticLivenessResourceResponse($input);
+
+        $this->assertEquals(self::SOME_LIVENESS_TYPE, $result->getLivenessType());
+        $this->assertNotNull($result->getImage());
+        $this->assertInstanceOf(MediaResponse::class, $result->getImage());
```

Comment:
shall we also assert the security id stored inside the media?

Reply:
Yes, I will add now.